src/Backend/Modules/ContentBlocks/Engine/Model.php

```diff
@@ -22,6 +22,7 @@
  * @author Matthias Mullie <forkcms@mullie.eu>
  * @author Dieter Vanden Eynde <dieter.vandeneynde@netlash.com>
  * @author Jeroen Desloovere <jeroen@siesqo.be>
+ * @author Jelmer Prins <jelmer@sumocoders.be>
```

Comment:
Still find it strange that we do this credits thing in Fork, while we can do git blame :P ? If we would have 100's of collaborators on git this would be funny to see it grow